ledger/participant-integration-api/src/main/resources/db/migration/postgres-appendonly/V117__vacuum_analyze.sql

```diff
@@ -0,0 +1,4 @@
+--  Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+--  SPDX-License-Identifier: Apache-2.0
+
+VACUUM ANALYZE;
```

Comment:
Is this the last step in the chain of migrations that implement the new ACS queries?

Reply:
exactly